src/apps/media/src/app-revamp/components/Thumbnail/index.tsx

```diff
@@ -80,17 +92,51 @@ export const Thumbnail: FC<ThumbnailProps> = ({
     case "gif":
     case "svg":
     case "webp":
+      return (
+        <Card sx={CardStyle} elevation={0} onClick={onClick}>
+          <Box
+            sx={{
+              py: imageOrientation === "horizontal" && 1,
+              px: imageOrientation === "vertical" && "auto",
+              boxSizing: "border-box",
+              maxHeight: "150px",
+              minHeight: "150px",
+              overflow: "hidden",
+            }}
+          >
+            <CardMedia
+              component="img"
+              ref={imageEl}
+              onLoad={handleImageLoad}
+              data-src={src}
+              image={src}
+              loading="lazy"
+              sx={{
+                objectFit:
+                  imageOrientation === "horizontal" ? "fill" : "contain",
+                maxHeight: "inherit",
```

Comment:
However, upon checking, the 160px includes the padding for horizontal images and not just the image itself. If that's the case, maybe it would be better if I just put the 160px on the parent Box?